storage-common/src/lib.rs

```diff
@@ -6,3 +6,11 @@ pub enum RequestParams {
     /// S3 bucket and keys.
     S3((String, Vec<String>)),
 }
+
+/// Initialize the logger for tests
+pub fn init_logger() {
+    let _ = env_logger::builder()
+        .filter_level(log::LevelFilter::Info)
+        .is_test(true)
```

Comment:
What's the difference between `true` and `false` here?

Reply:
https://docs.rs/env_logger/latest/env_logger/struct.Builder.html#method.is_test